### DISCUSSION TITLE
So confused! Loaded up WinUtils to see a new version, but Co-Pilot and FSO were already disabled??

### DISCUSSION THREAD

**Discussion by slikvik55** [ChrisTitusTech/winutil#2084] [Q&A]
Not sure what happened here. I've only run WinUtil once about a week ago and certainly didn't tick any of the Advanced Tweaks. 

Anyway, run it again today and it appeared to be a new version. Nice.
When I went to Tweaks and clicked _Get Installed_, it showed Disable Co-Pilot, Disable Fullscreen Opt. and Adobe Debloat already ticked?? What was even more weird is Co-Pilot actually WAS disabled on my Laptop!

To summarize:

- Those options weren't even there the last time I ran Winutil (I think)
- Co-Pilot WAS working on my Laptop (although I hardly use it so don't know when it disappeared).
- Even if those options were there I wouldn't have done the Advanced Tweaks anyway because of the warnings.

Apart from me going crazy, can anyone think why this might have happened?

**Comment by ModernTTY** [comment]
Well I can explain the Copilot thing, it seems like Microsoft is disabling Copilot for some users(usually for Windows insiders on 24H2) because of bugs.
